items.md

```diff
@@ -57,7 +57,68 @@ Exposed links:
 * owningCollection: the collection where the item belong to
 * templateItemOf: the collection that have the item as template
  
+## Patch operations
 
+### Replace
+The replace operation allows to replace *existent* information with new one. Attempt to use the replace operation to set not yet initialized information must return an error. See [general errors on PATCH requests](patch.md)
+
+To withdraw an item, `curl --data '{[ { "op": "replace", "path": "withdraw"}]}' -X PATCH ${dspace7-url}/api/core/items/${item-uuid}`.  The withdraw operation also requires an Authorization header.
```

Comment:
Thanks for the example. I see what you mean.  Will update the contract here before I update the code.

Reply:
Quick follow up question. 

The issue says that the request should return the 422 error code if the client try to withdrawn a NOT archived item. Does this refer to the "inArchive" flag in the item data?  

I notice that withdraw (and reinstate) actions toggle the inArchive flag.  For example, setting withdraw = true will result in inArchive = false. My initial idea was that requesting to withdraw and already withdrawn item would result in 442 (because the inArchive flag is false). I suspect this idea is incorrect.

Once I understand the requirement better I'll also update the code PR to do the right thing.

Reply:
You are right withdrawn items have the inArchive flag to false but this is not the only situation where the inArchive flag is false. InProgress submissions (workspaceitem, workflowitem) also have the inArchive flag to false.
Trying to withdrawn an item that is already withdrawn should succeed (it is a no-op)